lib/performance_platform.rb

```diff
@@ -11,56 +11,49 @@ def initialize(token = nil)
       @headers = { 'Authorization' => "Bearer #{token}",
                    'Content-Type' => 'application/json' }
     end
-  end
 
-  class TransactionsByChannel < Base
-    def submit_transactions(count, timestamp = Time.zone.now.iso8601, period = 'day')
+    def submit(data, timestamp = Time.zone.now.iso8601, period = 'day')
       HTTParty.post("#{PP_DOMAIN}#{transaction_endpoint}",
-                    body: data(timestamp, period, count),
+                    body: data_map(timestamp, period, data),
                     headers: headers)
     end
+  end
 
+  class TransactionsByChannel < Base
     private
 
     def transaction_endpoint
       '/data/teaching-jobs-job-listings/transactions-by-channel'
     end
 
-    def data(timestamp, period, count)
+    def data_map(timestamp, period, data)
       { _timestamp: timestamp,
         service: 'teaching_jobs_listings',
         channel: 'digital',
-        count: count,
+        count: data,
```

Comment:
We know what the data is here. I think it would be clearer to keep it named `count`.

Reply:
👍